pkgs/development/python-modules/pynacl/default.nix

```diff
@@ -10,12 +10,12 @@
 
 buildPythonPackage rec {
   pname = "pynacl";
-  version = "1.3.0";
+  version = "1.4.0";
 
   src = fetchPypi {
     inherit version;
     pname = "PyNaCl";
-    sha256 = "0c6100edd16fefd1557da078c7a31e7b7d7a52ce39fdca2bec29d4f7b6e7600c";
+    sha256 = "01b56hxrbif3hx8l6rwz5kljrgvlbj7shmmd2rjh0hn7974a5sal";
```

Comment:
Is this the correct sha256?  According to https://pypi.org/project/PyNaCl/1.4.0/ it should be `54e9a2c849c742006516ad56a88f5c74bf2ce92c9f67435187c3c5953b346505`...

Reply:
it's the same "sha", just expressed in a different base:
```
$ nix-hash --to-base16 --type sha256 01b56hxrbif3hx8l6rwz5kljrgvlbj7shmmd2rjh0hn7974a5sal
54e9a2c849c742006516ad56a88f5c74bf2ce92c9f67435187c3c5953b346505
```

Reply:
Thanks, TIL, just read the relevant section in the nixpkgs manual now :+1: